README.md

```diff
@@ -17,3 +17,52 @@ How to run
   * add db password for reporting username
   * add utas credentials in order to mount sheryl
 * 2) run ```./report/reporting_script.sh```
+
+
+Automatic reporting & notification
+=========
+
+For automatic reporting, the following software are required:
+
+1.sendmail
+2.ssmtp
+
+The conditions required are:
+
+1. Setup a cronjob to call `trigger_reporting_and_email.sh`
+2. Ssmtp is properly configured
+3. You can execute the reporting script.
+4. The Sheryl folder is always mounted.
+5. You edit trigger_reporting_and_email.sh to send email to right persons.
+
+Use The wrapper `trigger_reporting_and_email.sh` will trigger the reporting_script.sh, the checker and a sendmail job.
```

Comment:
just looking at the requirements. Since this script is not running on any production machine, but people's laptop, I don't think the email feature should be mandatory. It's great for you to be able to do it automatically, but it would be better if this was optional, so other PO's can run this script on their machine on your abscence without having to spend time nerding around ssmtp. Just my 2 cents.

Also for the ```You edit trigger_reporting_and_email.sh to send email to right persons.```. might as well have a configuration file dealing with this.

Reply:
All is optional - only changes affecting reporting is disable the sudo mount call and copy figure inside the r script, which should not affect runtime.